tests/unit/test_footprinter.cc

```diff
@@ -177,15 +177,6 @@ TEST_F(TestFootprinter, test_fundamental_types) {
     );
     delete [] arr;
   }
-
-  {
-    int arr[] = {1, 2, 3, 4, 5};
-    EXPECT_EQ(
-      checkpoint::getMemoryFootprint(arr),
-      sizeof(arr)
-      // actually: sizeof(&arr) + sizeof(arr)
-    );
-  }
```

Comment:
UBSAN found an error here:
```
reference binding to misaligned address 0x000200000001 for type 'int', which requires 4 byte alignment
```


`dispatch::with()` does `auto val = cleanType(&target);` which eventually does `std::decay` on C-array type. Simplifying - it casts it to a raw pointer, making serialization of an array not possible. That's why I'm removing this test.